pkg/utils/k8sutil.go

```diff
@@ -53,7 +53,8 @@ import (
 
 const (
 	controllerImage = "bitnami/kubeless-controller"
-	kafkaImage      = "wurstmeister/kafka"
+	kafkaImage      = "bitnami/kafka"
```

Comment:
@sebgoa Do you think we should also replace the flag `--controller-image` by a local ENV ? Just for consistency purpose.